huaweicloud/services/dws/resource_huaweicloud_dws_cluster.go

```diff
@@ -0,0 +1,574 @@
+package dws
+
+import (
+	"context"
+	"time"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/dws/cluster"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
+)
+
+func ResourceDwsCluster() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDwsClusterCreate,
+		ReadContext:   resourceDwsClusterRead,
+		DeleteContext: resourceDwsClusterDelete,
+		UpdateContext: resourceDwsClusterUpdate,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
+			Delete: schema.DefaultTimeout(60 * time.Minute),
+			Update: schema.DefaultTimeout(60 * time.Minute),
```

Comment:
All Timeouts are 60m, better to use an appropriate value, and need to update docs as well.

Reply:
OK, docs has been updated.  All operation time exceeds 40 minutes, so keep it as it is